webxdc.d.ts

```diff
@@ -60,6 +60,26 @@ type SendOptions =
       text: string;
     };
 
+/**
+ * A listener for realtime data.
+ */
+export class RealtimeListener {
+  listener: (data: Uint8Array) => void
+  active: boolean
+  trashed: boolean
```

Comment:
is one of these private? or "readonly"?

Reply:
already marked locally

Reply:
just tried it out in the ts playground, seems like typescript needs an explicit `private` or `protected` keyword, it is not like rust where everything is private by default. maybe there is a compiler flag for this, but I wouldn't bet on that.